packs/network-flow-devices/config.yml

```diff
@@ -0,0 +1,47 @@
+# Name of the pack (required)
+name: network-flow-devices-pack
+
+# Displayed in the UI
+title: Network Flow Devices Pack
```

Comment:
should the title refer to Kentik or are we positioning this as New Relic functionality?

Reply:
@polfliet - we have almost no mentions of Kentik in our public site or docs with the exception of things like *"ktranslate was created by our partners at Kentik"*

Since these packs were created and managed internally to NR; I don't think we need to name them here; I do think it makes sense to leave them as a keyword for discoverability

Reply:
@thezackm that makes sense to me to use the keywords for this purpose, feel free to reject my suggestions above if that is the approach.